ci/docker/Dockerfile-nighthawk-benchmark

```diff
@@ -8,7 +8,7 @@ WORKDIR /usr/local/bin/benchmarks
 
 COPY benchmarks /usr/local/bin/benchmarks/
 
-RUN apk add --no-cache docker=20.10.3-r0 openrc=0.42.1-r19 python3>=3.8.7-r0
+RUN apk add --no-cache docker>=20.10.3-r0 openrc>=0.42.1-r19 python3>=3.8.7-r0
```

Comment:
@mum4k I think this change deflakes the docker benchmark image build runs.
/cc @abaptiste